packages/builder/src/components/design/settings/controls/DataSourceSelect/DataSourceSelect.svelte

```diff
@@ -55,6 +55,7 @@
     label: m.name,
     tableId: m._id,
     type: "table",
+    datasource: $datasources.list.find(ds => ds._id === m.sourceId || m.datasourceId),
```

Comment:
Do tables have a `datasourceId` property? I thought that was just queries that have that but I could be wrong.

Reply:
they don't, actually I can remove this because I planned to do it for queries too then realised it's not as relevant. Will do that in next PR!